lib/is-cjs-esm-bridge.js

```diff
@@ -1,10 +1,17 @@
 module.exports = ({ functions }) => {
   // https://github.com/nodejs/node/blob/v12.1.0/lib/internal/modules/esm/create_dynamic_module.js#L11-L19
+  /** * / I don't think this is correct ...
   return functions.length === 3 &&
   functions[0].functionName === '' &&
   functions[0].isBlockCoverage === true &&
   functions[1].functionName === 'get' &&
   functions[1].isBlockCoverage === false &&
   functions[2].functionName === 'set' &&
-  functions[2].isBlockCoverage === true
+  functions[2].isBlockCoverage === false
+  /** */
+  return functions.length === 2 &&
+  functions[0].functionName === '' &&
+  functions[0].isBlockCoverage === true &&
+  functions[1].functionName === 'module.exports' &&
+  functions[1].isBlockCoverage === true
```

Comment:
[Lines 114 of reports.js](https://github.com/bcoe/c8/blob/main/lib/report.js#L114)  calls this function to see if a common js or cjs file has been imported into ECMAScript or esm file.  The issue was the previous block of code, only ever returned a boolean false, and  [lines of 115 - 125 report.js](https://github.com/bcoe/c8/blob/main/lib/report.js#L115-L125) were not being reached.  A test case in the file [integrations.js](https://github.com/bcoe/c8/blob/main/test/integration.js#L280-L295) covers the use case for this particular item.  

Below is an example of the v8ScriptCov variable on [line 114](https://github.com/bcoe/c8/blob/main/lib/report.js#L114)

```{
  scriptId: '0',
  url: '/home/tmckeown/codez/c8/test/fixtures/export.cjs',
  functions: [
    { functionName: '', ranges: [Array], isBlockCoverage: true },
    {
      functionName: 'module.exports',
      ranges: [Array],
      isBlockCoverage: true
    }
```

Reply:
I believe the logic for handling importing ESM/CJS was changed several times during the the Node 12 release line. It's possible this logic worked as advertised in `12.1.0`, but in the current stable release of Node 12 is a noop.

I don't think we should remove the logic, since it fixed a real user issue at the time, but perhaps we can add a note:

// Remove this logic when we make Node 14 our minimum runtime version

Reply:
Ok.